src/views/BrowseView/BrowseView.style.ts

```diff
@@ -20,7 +25,7 @@ export const StyledCategoryPicker = styled(CategoryPicker)<IsAtTop>`
   top: ${NAVBAR_HEIGHT}px;
   padding: ${sizes(5)} var(--global-horizontal-padding) ${sizes(2)};
   margin: 0 calc(-1 * var(--global-horizontal-padding));
-  background-color: ${colors.black};
+  background-color: ${(props) => (props.isAtTop ? colors.transparent : colors.black)};
```

Comment:
Since we're changing that, I think it could be nice to add a `transition` for background so the change is more seamless